parm/land/letkfoi/letkfoi.yaml

```diff
@@ -28,7 +28,7 @@ background:
      template:
        datetime: '{{ current_cycle | to_isotime }}'
        filetype: fms restart
-       state variables: [snwdph,vtype,slmsk]
+       state variables: [$(SNOWDEPTHVAR) ,vtype,slmsk]
```

Comment:
Didn't we land on the assumption that we would just support the fractional grid and not worry about backwards compatibility or am I misremembering?

Reply:
The YAML in ush/land is fixed to snodl.

Reply:
The current changes in the `letkfoi.yaml` support both the fractional grid and non fractional grid. I have conducted some tests to confirm that the above changes can be used in the global-workflow for both fractional and non fractional grids. 

The YAML file `ush/land/genYAML_output_letkfoi.yaml` just supports the ctest in the GDASApp, while the above `letkfoi.yaml` will be used in the global-workflow. Therefore, the variable name in the `ush/land/genYAML_output_letkfoi.yaml` is hard-coded to `snodl`.